pixi.toml

```diff
@@ -0,0 +1,13 @@
+[project]
+authors = ["Scientific Python"]
+channels = ["conda-forge"]
+description = "Environment for the scientific-python/upload-nightly-action GitHub action"
+name = "upload-nightly-action"
+platforms = ["linux-64", "osx-64", "osx-arm64"]
```

Comment:
Adding the macOS platforms as though I assume that people will be using this following a `cibuildwheel` workflow on Linux, providing the extra platforms in the lock file allows for this action to be run on macOS runners as well.